### ISSUE TITLE
'sequel' is not a valid integration

### ISSUE THREAD

**Issue by steveh** [DataDog/dd-trace-rb#514]
As of v0.14.0, the Sequel integration has stopped working.

```ruby
Datadog.configure do |c|
  c.use :sequel
end
```

```bash
	 3: from .gem/ruby/2.5.1/gems/ddtrace-0.14.0/lib/ddtrace.rb:38:in `configure'
	 2: from config/initializers/datadog.rb:11:in `block in <top (required)>'
	 1: from .gem/ruby/2.5.1/gems/ddtrace-0.14.0/lib/ddtrace/configuration.rb:26:in `use'
.gem/ruby/2.5.1/gems/ddtrace-0.14.0/lib/ddtrace/configuration.rb:56:in `fetch_integration': 'sequel' is not a valid integration. (Datadog::Configuration::InvalidIntegrationError)
```

Inspecting the Registry object shows many other integrations are present, but not Sequel

**Comment by delner** [comment]
I suspect the issue might be that we're missing `require 'ddtrace/contrib/sequel/integration'` in our `ddtrace.rb` file. This integration was recently upgraded. Can you try adding `require 'ddtrace/contrib/sequel/integration'` at the top of your configuration file to see if that works?

**Comment by delner** [comment]
Replicated this by commenting the explicit require we had in our tests. #516 should fix the issue. Will aim to ship this in the next bugfix release.

**Comment by steveh** [comment]
Thanks!

**Comment by steveh** [comment]
fixed by #516
